### ISSUE TITLE
MEME and aBSREL – Error in processing jobs

### ISSUE THREAD

**Issue by Pernini** [veg/hyphy#743]
I am continuously getting error messages when I try to run MEME (and as of today also for aBSREL) on 9-sequence alignments of various genes (running alignments separately). The jobs seems to hang and eventually the error message "Could not contact server for job status updates" appears. Refreshing the webpage makes the runtime clock restart from 0. I have also gotten the message "Error processing your job".

I've previously used alignments in fasta format as input, of which one managed to complete (9 sequences, 384 sites). I now want to run the analysis with gene trees that I have produced and have created nexus files containing both alignment and tree. None of the jobs I have started using the nexus files have managed to complete and they all receive the same aformentioned error messages. To be fair, the longest runtime is currently 26 hrs, but the continuous error messages is of concern.

Example: MEME with  Ticket Number 868602.silverback has no Log output, the error message "could not contact server ..()" keep appearing but the runtime keeps counting. And the job cannot be seen on the Job Queue.

I do not get warnings about the nexus file format being wrong, and a resubmission of the alignment that managed to complete now also receives error messages. 

Basically I'm not sure if this is a server issue, but something seems off at least.

**Comment by stevenweaver** [comment]
Dear @Pernini, 

This issue should now be resolved. The error occurred due to excessive logging to standard output in MEME analyses, which was causing bandwidth issues with our services. 

Best,
Steven
